cmd/oras/fetch-manifest.go

```diff
@@ -31,26 +35,28 @@ type getManifestOptions struct {
 	option.Remote
 
 	targetRef string
-	output    string
-	pretty    bool
+	raw       bool
+	indent    int
+	platform  string
+	mediaType string
 }
 
 func fetchManifestCmd() *cobra.Command {
 	var opts getManifestOptions
 	cmd := &cobra.Command{
-		Use:   "fetch-manifest <name:tag|name@digest>",
+		Use:   "fetch-manifest [flags] <name:tag|name@digest>",
 		Short: "[Preview] Fetch manifest of the target artifact",
 		Long: `[Preview] Fetch manifest of the target artifact
 ** This command is in preview and under development. **
 
 Example - Get manifest:
   oras get-manifest localhost:5000/hello:latest
 
-Example - Get manifest and save to manifest.json:
-  oras get-manifest -output manifest.json localhost:5000/hello:latest
+Example - Get manifest with specified media type:
+  oras get-manifest --media-type 'application/vnd.oci.image.manifest.v1+json' localhost:5000/hello:latest
 
-Example - Get manifest with prettified json result:
-  oras get-manifest --pretty localhost:5000/hello:latest
+Example - Get manifest with raw json result:
+  oras get-manifest --raw localhost:5000/hello:latest
```

Comment:
Is the command get-manifest or fetch-manifest. 
Lesser characters the better IMHO. 
can we just go for manifest get and put. 
this way if someone downloads a manifest and adds and annotation, they can then upload the updated manifest as well. 

/cc @FeynmanZhou

Reply:
Now in this PR I am aligning all the naming to fetch-manifest. 

I suggest we use `fetch` command and placing all the feature provided in this PR into its sub-command `manifest`. So the command would look like
```
oras fetch manifest localhost:5000/hello:latest
```

UX like `oras manifest get localhost:5000/hello:latest` is not unified from what we have already provided since current all commands are verbs (push, pull, attach, discover)

Reply:
I would still recommend  for get and put since they are shorter and more user friendly. The CLI is about UX which can and may diverge from the go libraries. Check with @FeynmanZhou on the usability of this command. 

Similar to pull and push these are dockerism which don’t have equivalence in the libraries.

Reply:
Let me discuss with @FeynmanZhou. BTW `git` also uses `fetch` on status updating (so common developers should have muscle memory of typing this word :) ).

Reply:
Btw docker also has a manifest command eve though they have first level verbs. Like images etc. please also make sure we have a provision to write a manifest even if we don’t mAke the command immediately.

Reply:
> Btw docker also has a manifest command eve though they have first level verbs. Like images etc. please also make sure we have a provision to write a manifest even if we don’t mAke the command immediately.

So do crane. Will discuss all that with @FeynmanZhou when @shizhMSFT is back next week.

Reply:
Updated to `manifest fetch` and using `get` as an alias